.github/workflows/build-client-packages.yml

```diff
@@ -0,0 +1,126 @@
+name: Build client packages
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  build-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get the version number for the packages
+        id: get_version
+        # If this is a tag, use the tag name (e.g. v1.2.3) without v as version number.
+        # Otherwise, just use 0.0.
+        run: |
+          VERSION=0.0
+          REF_NAME=${{ github.ref }}
+          [[ $REF_NAME == refs/tags/v* ]] && VERSION=${REF_NAME/refs\/tags\/v/}
+          echo ::set-output name=version::${VERSION}
+
+      # The next step uses a custom Ansible inventory, and due to that it cannot find
+      # the group_vars folder inside the inventory folder. This symlink fixes that.
+      - name: Make symlink to group_vars
+        run: ln -s inventory/group_vars
+
+      - name: Prepare package source
+        uses: roles-ansible/check-ansible-debian-stretch-action@master
+        with:
+          targets: "./prepare-client-packages.yml"
+          hosts: "localhost"
+
+      - name: Build RPM package
+        id: build-rpm
+        uses: kentik/pkg@v1.0.0-rc6
+        with:
+          name: cvmfs-config-eessi
+          version: ${{ steps.get_version.outputs.version }}
+          arch: x86_64
```

Comment:
Why is this `x86_64`, there's no binary stuff in the package? Can it be `noarch`?

Reply:
Yeah, just made a comment about this. I don't like it either, but this tool doesn't support noarch. I'll have a look at https://github.com/jordansissel/fpm instead, which also has a Github action.

Reply:
EasyBuild also supports `fpm` as packaging tool, it's used pretty commonly...

Reply:
It's now using fpm, and building for noarch/all.